src/random.jl

```diff
@@ -64,25 +70,29 @@ multi-normal with the covariance of any pair of payoffs equal to
   Nash Equilibria in Random Games," Games and Economic Behavior
   (2000), 274-293.
 """
-function covariance_game{N}(nums_actions::NTuple{N,Int}, rho::Real)
+function covariance_game{N}(rng::AbstractRNG, nums_actions::NTuple{N,Int},
+                            rho::Real)
     if N <= 1
         throw(ArgumentError("length of nums_actions must be at least 2"))
     end
 
-    if !(-1 / (N - 1) < rho < 1)
+    if !(-1 / (N - 1) <= rho <= 1)
         lb = (N == 2) ? "-1" : "-1/$(N-1)"
-        throw(ArgumentError("rho must be in ($lb, 1)"))
+        throw(ArgumentError("rho must be in [$lb, 1]"))
     end
 
     mu = zeros(N)
     C = fill(rho, (N, N))
```

Comment:
The notation `C` follows `Distributions.jl`. Should we change it to `Sigma` to keep consistent with [MVNSampler](https://github.com/QuantEcon/QuantEcon.jl/blob/20a7e621792cc68bf49712354d0218cd61a0ded0/src/sampler.jl#L9)?